infra/outputs.tf

```diff
@@ -47,3 +46,7 @@ output "usage" {
     Password: ${google_secret_manager_secret_version.django_admin_password.secret_data}
     EOF
 }
+
+output "region" {
```

Comment:
nit: we prefer not to redirect inputs as outputs whenever possible. If this was for usage in test, you can just hardcode the region in test as a const.